contracts/pcv/curve/StableSwapOperatorV1.sol

```diff
@@ -83,6 +94,27 @@ contract StableSwapOperatorV1 is PCVDeposit {
         _usdt = IStableSwap3(_curve3pool).coins(2);
     }
 
+
+    /// @notice set the minimum ratio threshold for a valid reading of restistant balances
+    function setMinRatio(uint256 _minimumRatioThreshold) public onlyGovernor {
+        require(_minimumRatioThreshold < maximumRatioThreshold, "Min ratio must be less than max ratio");
+        require(_minimumRatioThreshold >= 0.01e18, "Min ratio must be at least 1%.");
+        minimumRatioThreshold = _minimumRatioThreshold;
+    }
+
+    /// @notice set the maximum ratio threshold for a valid reading of resistant balances
+    function setMaxRatio(uint256 _maximumRatioThreshold) public onlyGovernor {
+        require(_maximumRatioThreshold > minimumRatioThreshold, "Max ratio must be greater than min ratio");
+        require(_maximumRatioThreshold <= 100e18, "Max ratio cannot be above 100x.");
+        maximumRatioThreshold = _maximumRatioThreshold;
+    }
+
+    /// @notice set both min & max ratios
+    function setRatios(uint256 _minimumRatioThreshold, uint256 _maximumRatioThreshold) public onlyGovernor {
+        setMinRatio(_minimumRatioThreshold);
+        setMaxRatio(_maximumRatioThreshold);
```

Comment:
The order here matters somewhat as there is a dependency between the require statements. TBH we probably don't need this method